examples/auth_example/auth_example_server/migrations/serverpod/20231114162733/migration.sql

```diff
@@ -143,6 +143,19 @@ CREATE TABLE "serverpod_method" (
 -- Indexes
 CREATE UNIQUE INDEX "serverpod_method_endpoint_method_idx" ON "serverpod_method" USING btree ("endpoint", "method");
 
+--
+-- ACTION CREATE TABLE
+--
+CREATE TABLE "serverpod_migrations" (
+    "id" serial PRIMARY KEY,
+    "module" text NOT NULL,
+    "version" text NOT NULL,
+    "timestamp" timestamp without time zone
+);
```

Comment:
Yes, creating the migration is now part of the first migration applied, so we should be OK for new projects being created (since we apply migrations during project creation) and also for project being migrated when we set up our "coming to 1.2" workflow.

If we try and read what migrations are installed and the table doesn't exist (since that is the first thing we do on startup) we just assume that no migrations are installed. This can be the case when we have a new project that has not been migrated yet or when upgrading a project from pre 1.2 to 1.2.

The point is that the migration table is no different than the other "required" tables for running Serverpod. As far as I understand, we really shouldn't need another flow for it.

Reply:
(And this is the serverpod migration copied to the auth example project that we do when we generate a project, so it is only the serverpod migration/package that is responsible for creating the migration)